aev/src/main/java/com/fingerprintjs/android/aev/raw_signal_providers/SensorsDataCollector.kt

```diff
@@ -68,5 +90,20 @@ internal class SensorsDataCollectorImpl(private val sensorManager: SensorManager
     }
 }
 
-private const val NUMBER_OF_SENSOR_VALUES = 30
-private const val TIMEOUT_IN_MILLIS = 1500L
\ No newline at end of file
+internal class SensorsDataCollectorBuilder(
+    private val sensorManager: SensorManager,
+) {
+    private var config: Config = Config.DEFAULT
+
+    fun withConfig(config: Config): SensorsDataCollectorBuilder {
+        this.config = config
+        return this
+    }
+
+    fun build(): SensorsDataCollectorImpl {
```

Comment:
It should return `SensorsDataCollector`, not `Impl`, right?

Reply:
Yep, thanks